### PR TITLE
{bio}[foss/2023a] Single-cell-python-bundle v2024.02 w/ CUDA 12.1.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/scvi-tools/scvi-tools-1.1.2-foss-2023a-CUDA-12.1.1.eb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        'checksums': ['6f172d5449aca15afd6c646851f4e31e02c598d553a667e38cafa997cfec55fe'],
+    }),
+    ('orbax_checkpoint', '0.5.15', {
+        'modulename': 'orbax.checkpoint',
+        'checksums': ['15195e8d1b381b56f23a62a25599a3644f5d08655fa64f60bb1b938b8ffe7ef3'],
+    }),
+    ('tensorstore', '0.1.60', {
+        'checksums': ['88da8f1978982101b8dbb144fd29ee362e4e8c97fc595c4992d555f80ce62a79'],
+    }),
+    ('flax', '0.8.4', {
```
Comment (boegel) [inline easybuild/easyconfigs/s/scvi-tools/scvi-tools-1.1.2-foss-2023a-CUDA-12.1.1.eb:51]: This overlaps with #21039, which should get merged first...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3302.joltik.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, 1 x NVIDIA Tesla V100-SXM2-32GB, 545.23.08, Python 3.6.8
See https://gist.github.com/boegel/19f79ed944d7969ce26cbc7677b2ea22 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3-a100

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3905.accelgor.os - Linux RHEL 8.8, x86_64, AMD EPYC 7413 24-Core Processor, 1 x NVIDIA NVIDIA A100-SXM4-80GB, 545.23.08, Python 3.6.8
See https://gist.github.com/boegel/7c8a9db5fcc618108144c70b688fe75e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3300.joltik.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, 1 x NVIDIA Tesla V100-SXM2-32GB, 545.23.08, Python 3.6.8
See https://gist.github.com/boegel/7bffbbbd8786a53a97c156590ef3c94a for a full test report.

**Review by laraPPr — APPROVED** [review]
lgtm

**Comment by laraPPr** [comment]
Going in, thanks @boegel!

**Pull request merged**